organization.go

```diff
@@ -183,7 +183,9 @@ type OrganizationPermissions struct {
 	CanCreateTeam               bool `jsonapi:"attr,can-create-team"`
 	CanCreateWorkspace          bool `jsonapi:"attr,can-create-workspace"`
 	CanCreateWorkspaceMigration bool `jsonapi:"attr,can-create-workspace-migration"`
+	CanDeployNoCodeModules      bool `jsonapi:"attr,can-deploy-no-code-modules"`
 	CanDestroy                  bool `jsonapi:"attr,can-destroy"`
+	CanManageNoCodeModules      bool `jsonapi:"attr,can-manage-no-code-modules"`
```

Comment:
These are exposed through the [organization serializer](https://github.com/hashicorp/atlas/blob/3c5c0d829512063b33afe7d1984b04158accc9f6/app/serializers/api/v2/organization_serializer.rb#L284-L285)